planemo/lint.py

```diff
@@ -20,8 +20,16 @@ def build_lint_args(ctx, **kwds):
         skip = ctx.global_config.get("lint_skip", "")
         if isinstance(skip, list):
             skip = ",".join(skip)
-
     skip_types = [s.strip() for s in skip.split(",")]
+
+    for skip_file in kwds.get("skip_file", []):
+        with open(skip_file) as f:
+            for line in f.readlines():
+                line = line.strip()
+                if not line or line.startswith("#"):
+                    continue
+                skip_types.append(line)
```

Comment:
Can we make sure these are valid ? I'd love to see a test as well, can't really imagine what that will look like

Reply:
Good idea. I would add a function to list the available linters over here: https://github.com/galaxyproject/galaxy/pull/17081 

Lets see if I find out how to add a test here.